providers/universal-provider/src/UniversalProvider.ts

```diff
@@ -138,6 +139,32 @@ export class UniversalProvider implements IUniversalProvider {
     return await this.pair(opts.pairingTopic);
   }
 
+  public async authenticate(opts: AuthenticateParams) {
+    if (!this.client) {
+      throw new Error("Sign Client not initialized");
+    }
+    this.setNamespaces(opts);
+    await this.cleanupPendingPairings();
+
+    const { uri, response } = await this.client.authenticate({
+      ...opts,
+    });
```

Comment:
No need to spread here 
```suggestion
    const { uri, response } = await this.client.authenticate(opts);
```